tasks-reference/pam/text.xml

```diff
@@ -163,25 +145,15 @@ completely non-portable. It is not used in all the implementations of Linux-PAM
 
```

Comment:
The paragraph above talks about `other PAM implementations`. This should also be updated, no?